ksql-common/src/main/java/io/confluent/ksql/function/udf/Udf.java

```diff
@@ -21,6 +21,10 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * Used to signal that a method in a class that has the @UdfDescription annotation
+ * as a function that can be invoked.
```

Comment:
do you mean "is a function that can be invoked"?

Reply:
Maybe: "The {@code Udf} annotation on a method tells KSQL that this method should be exposed as UDF. The enclosing class must also be annotated with {@code UdfDescription}."